lib/client/weblogin.go

```diff
@@ -320,7 +320,7 @@ func SSHAgentLogin(ctx context.Context, login SSHLoginDirect) (*auth.SSHLoginRes
 		return nil, trace.Wrap(err)
 	}
 
-	re, err := clt.PostJSON(ctx, clt.Endpoint("webapi", "ssh", "certs"), CreateSSHCertReq{
+	re, err := clt.PostJSONWithFallback(ctx, clt.Endpoint("webapi", "ssh", "certs"), login.Insecure, CreateSSHCertReq{
```

Comment:
Why is this need? This should not be needed.

Reply:
It was an artefact of the testing environment I had set up. Definitely not needed.

Reply:
Removed.